tests/integration_python/test_exec.py

```diff
@@ -17,12 +17,12 @@ def test_concurrent_exec(pixi: Path, dummy_channel_1: str) -> None:
         futures = [
             executor.submit(
                 verify_cli_command,
-                [pixi, "exec", "-c", dummy_channel_1, "dummy-f"],
+                [pixi, "exec", "-c", dummy_channel_1, "--", "dummy-f"],
```

Comment:
The feature is not important to introduce a breaking change in our CLI.

I don't really understand though why this change is necessary, can you explain the problem to me?